### DISCUSSION TITLE
[Bug] 249 error typeerror This is an error with npm itself...

### DISCUSSION THREAD

**Discussion by ghost** [npm/feedback#512] [General]
Problem:
I'm a newb (lol)

Ok, ... NPM-related issue:
Failed installation of mammoth.js on LM19.3 with NPM stating there is an error with NPM itself; install log attached.

Commands Used:
git clone https://github.com/mwilliamson/mammoth.js.git
cd mammoth.js
make setup (failed)
npm install eslint mocha node-pty
npm install mammoth (failed)
errors; see npm-debug.log (attached)

Solution:
I don't have one. I did a search for a few of the error lines and I tried installing a few potential missing dependencies such as eslint, mocha, node-pty, but mammoth still fails with the same errors.

Inxi -S:
Host: [redacted]
Kernel: 5.4.0-80-generic x86_64
bits: 64
Desktop: Cinnamon 4.4.8
Distro: Linux Mint 19.3 Tricia

[npm-debug.log](https://github.com/npm/feedback/files/6919291/npm-debug.log)

**Comment by MylesBorins** [comment]
Hey @EnilNoliam . This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
